### DISCUSSION TITLE
Incorrect string value for "™"

### DISCUSSION THREAD

**Discussion by SearchDream** [PyMySQL/PyMySQL#1132] [Q&A]
**Describe the bug**
I am using Django with PyMySQL, when save the string which contain "™" an error message will come out. 
"1366, "Incorrect string value: '\\xF0\\x9F\\x98\\x89\\x0A\\x0A...' for column 'description' at row 1")"
PyMySQL v1.0.3 works for this string.

**Environment**
 - OS: [e.g. Windows, Linux]
 - Server and version: [e.g. MySQL 8.0.19, MariaDB]
 - PyMySQL version: v1.1.0

**Comment by methane** [comment]
you shoud use UTF-8 if you can not fix it by yourself.

**Reply by SearchDream** [reply]
Thanks for your reply, with the updated version of Django seems the issues gone.
